docs/guides/reproducible-build-verification.md

```diff
@@ -0,0 +1,118 @@
+# Reproducible build verification
+
+[Reproducible builds](https://reproducible-builds.org/) are crucial from both
+security and open-source perspectives because they allow anyone to verify that
+the compiled binary of a software package truly matches the original source
+code. This ensures that no tampering, such as inserting malicious code during
+the build process, has occurred.
+
+The most obvious and undisputable way of verifying build reproduction is
+comparing `sha256` or `md5` hashes of two given binaries. There are, however,
```

Comment:
maybe just say `hash`, `md5` may not be that `undisputable` anymore

Reply:
[Fixed](https://github.com/Dasharo/docs/pull/921/commits/936c81929769481e9bfc9db9e079a9d9c719569c#diff-e4434d45287f1a1685277fe23157c8a8613fe7209c8994b241a9c310469cce45R10)